tests/integration_tests/tests/agentless_tests/test_snapshot.py

```diff
@@ -254,17 +254,28 @@ def test_v_4_2_restore_snapshot_with_resource_availability(self):
         assert blueprints[2]['id'] == 'blueprint_3' and \
             blueprints[2]['visibility'] == 'global'
 
-    def test_v_4_3_encrypt_secrets_in_restore(self):
+    def test_v_4_3_restore_snapshot_with_secrets(self):
         """
         Validate the encryption of the secrets values for versions before 4.4.0
         """
-        snapshot_name = 'snap_4.3.0_with_secrets.zip'
+        self._test_secrets_restored('snap_4.3.0_with_secrets.zip')
+
+    def test_v_4_4_restore_snapshot_with_secrets(self):
+        """
+        Validate the restore of the secrets values for snapshot of 4.4.0
+        """
+        self._test_secrets_restored('snap_4.4.0_with_secrets.zip')
+
+    def _test_secrets_restored(self, snapshot_name):
         snapshot_path = self._get_snapshot(snapshot_name)
         self._upload_and_restore_snapshot(snapshot_path)
-        secrets = self.client.secrets.list(_include=['key'])
-        assert len(secrets) == 3
+
+        # Wait for the restart of cloudify-restservice to end
```

Comment:
Please add here a link to the code, or at least the name of the file and line number, so that people know where to look to understand the commit. Maybe expand the comment a bit as well.